task/deprecated-image-check/0.2/deprecated-image-check.yaml

```diff
@@ -60,7 +63,9 @@ spec:
           value: $(params.POLICY_NAMESPACE)
       script: |
         #!/usr/bin/env sh
+        set -euo pipefail
         source /utils.sh
+        trap cleanup ERR
 
```

Comment:
Shouldn't this script also have: 
` trap 'echo "Line:$LINENO"' ERR`
Since it uses pipe below? Line 85

Reply:
`cleanup` will add the error info to `TEST_OUTPUT`